dvc/config.py

```diff
@@ -234,7 +234,7 @@ class RelPath(str):
         "row_cleanup_quota": All(Coerce(int), Range(0, 100)),
     },
     # section for experimental features
-    "feature": {Optional("parametrization", default=False): Bool},
+    "feature": {Optional("parametrization", default=True): Bool},
```

Comment:
keeping it till the official release.

Reply:
Maybe, it'd be better if we allowed anything in here and don't complain.

Reply:
Yeah, could allow anything in this section. Someone is definitely using this option with viewer :smile: 

Let's add a comment in the code that this is no longer used. Aslo no reason to set default at all, since it is ignored anyway, right?

Reply:
@efiop, not really. This is still used, so that you could explicitly disable it.

Reply:
@skshetry Could you elaborate on a scenario for that? Debugging?

Reply:
No reason, just wanted to provide a way to disable for preview releases.

Reply:
I'll just remove those logics.